src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingSubmissionAndResultIntegrationTest.java

```diff
@@ -462,6 +464,42 @@ void shouldCreateSubmissionsForAllParticipationsOfExerciseAfterTestRepositoryCom
         }
     }
 
+    /**
+     * This tests if the submission is created for programming exercises without local setup
+     */
+
+    @Test
+    @WithMockUser(username = "instructor1", roles = "INSTRUCTOR")
+    void shouldCreateSubmissionWithoutLocalSetup() throws Exception {
+        assertThat(submissionRepository.findAll()).hasSize(0);
+        final var returnedSubmission = request.postWithResponseBody("/api" + SUBMISSIONS_SIMULATION + "/" + exerciseId, null, ProgrammingSubmission.class, HttpStatus.OK);
```

Comment:
Should return `201 Created` instead of `200 OK`.

Reply:
i changed it to 201